core/drivers/crypto/caam/hal/imx_6_7/hal_cfg.c

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2020-2021 NXP
+ */
+#include <caam_hal_cfg.h>
+
+void caam_hal_cfg_setup_nsjobring(struct caam_jrcfg *jrcfg)
+{
+	caam_hal_cfg_common_setup_nsjobring(jrcfg);
```

Comment:
Wouldn't it make more sense to define `caam_hal_cfg_setup_nsjobring()` as `__weak` instead? And same for `caam_hal_rng_instantiated()`.

Reply:
Good suggestion.